static/js/app.ts

```diff
@@ -1251,6 +1266,41 @@ function resetTurtleTarget() {
     return null;
 }
 
+function animateKeys(event: KeyboardEvent) {
+    const keyColors = ['#cbd5e0', '#bee3f8', '#4299e1', '#ff617b', '#ae81ea', '#68d391'];
+    const output = document.getElementById("output");
+
+    if (output !== null) {
+      let keyElement = document.createElement('div');
+      output.appendChild(keyElement);
+
+      keyElement.style.position = 'absolute';
+      keyElement.style.fontFamily = `system-ui,
+      -apple-system,BlinkMacSystemFont,
+      "Segoe UI",Roboto,
+      "Helvetica Neue",
+      Arial,"Noto Sans",
+      sans-serif,"Apple Color Emoji",
+      "Segoe UI Emoji","Segoe UI Symbol",
+      "Noto Color Emoji"`
+
+      keyElement.style.right = '20px';
+      keyElement.style.bottom = '10px';
+      keyElement.textContent = event.key;
+      keyElement.style.color = keyColors[Math.floor(Math.random() * keyColors.length)];
+
+      keyElement.style.transition = 'all 1.0s linear 0s';
+      keyElement.style.right = '15px';
+      keyElement.style.bottom = keyElement.offsetTop + 10 + 'px';
+      keyElement.style.fontSize = '50px';
+      keyElement.style.opacity = '0';
```

Comment:
Could we add all this styling to a css class and simply use `keyElement.addClass('...')`. Would make the code a bit more readable and enables us to use Tailwind for styling the element.

Reply:
I added a class _animate-keys_ and made use of _keyframes_ to make this possible. [Add animate-keys class to keys](https://github.com/hedyorg/hedy/pull/3668/commits/71af916c5cbe35732283e41cbe37957e9bb6fdcf) should fix this.